commons/pom.xml

```diff
@@ -108,19 +108,19 @@
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-annotations</artifactId>
-			<version>2.9.2</version>
+			<version>2.9.4</version>
```

Comment:
Can we move the jackson and guava dependencies up to the parent pom dependencies, or at least put the version into a property at the parent level since multiple poms use them?